dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -1255,6 +1256,15 @@ public void testAppToolCollections() throws Exception {
         assertTrue((collection.getEntries().stream().anyMatch(entry -> Objects.equals(entry.getId(), appTool.getId()))));
     }
 
+    /*
+     * TODO: reimplement
```

Comment:
Is there a ticket to follow up on this?

Reply:
Just making sure this isn't skipped over -- I don't see a recent issue on it (maybe there already was one?).